srcpkgs/busted/template

```diff
@@ -0,0 +1,57 @@
+# Template file for 'busted'
```

Comment:
`lua54-busted`? Or is there a specific reason why the lua 5.4 version should be handles special?

Reply:
It's a cli app so I am not too sure what the appropriate action is here (lua5- prefix or standalone name), so uhhh I did both

I am open to changing it /shrug

Reply:
in retrospect, doesn't make sense to have a lua54-busted and a busted package